rest-server/src/components/user/userController.js

```diff
@@ -0,0 +1,24 @@
+import dataBase from '../../config/db';
+import UserHelper from './userHelpers'
+import { error } from '../../lib/log';
+
+export default class UserController {
```

Comment:
Same as before. I'm open to hear why we should do it this way but I'm just not finding much of a reason to write it with classes. 

Mirroring front end code? Not sure